robot_specific_config/abb_irb1200_support/launch/load_irb1200_7_70.launch

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" ?>
```

Comment:
If there's a reason to keep the ROS1 launch files, we still may want to remove the IRB1200 7 70 launch files as they're for a robot we don't support.